### PR TITLE
notes and rooms

### PR DESCRIPTION
**Pull request by Brayn7**
created migrations models and controllers for notes and rooms.

### CODE REVIEW

File: ibackend/app/Note.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+ues Illuminate\Database\Eloquent\SoftDeletes;
```
Comment (stockdalemark1978) [inline ibackend/app/Note.php]: says "ues" instead of use

File: ibackend/database/migrations/2017_07_17_182052_create_notes_table.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notes', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('text');
+            $table->integer('user_id');
+            $table->foriegn('user_id')->references('id')->on('users');
```
Comment (stockdalemark1978) [inline ibackend/database/migrations/2017_07_17_182052_create_notes_table.php]: should be foreign, not foriegn

**Comment by doubldragon** [comment]
+1

File: ibackend/app/Room.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{
+    use SoftDeletes;
+
+    protected $dates = ['deleted_at'];
+
+    public function notes() {
+    	return $this->morphMany('App\Note', 'noteable');
+    }
+
+    public fucntion sites() {
```
Comment (dhopwood) [inline ibackend/app/Room.php]: function instead of fucntion

**Comment by dhopwood** [comment]
+1

**Comment by jctu225** [comment]
+1

**Pull request merged**